cosmwasm/contracts/crosschain-swaps/src/checks.rs

```diff
@@ -1,7 +1,19 @@
 use cosmwasm_std::{Addr, Deps};
 use regex::Regex;
 
-use crate::{state::CHANNEL_MAP, ContractError};
+use crate::{
+    state::{CHANNEL_MAP, CONFIG},
+    ContractError,
+};
+
+pub fn check_is_contract_owner(deps: Deps, sender: Addr) -> Result<(), ContractError> {
```

Comment:
Should we add a unit test for this function?

Reply:
added